project/BalboaAgent.scala

```diff
@@ -4,28 +4,16 @@ import com.typesafe.sbt.packager.debian
 import com.typesafe.sbt.packager.debian.DebianPlugin.autoImport._
 import sbt.Keys._
 import sbt._
-import sbtassembly.Plugin.AssemblyKeys._
 
 object BalboaAgent {
-  lazy val settings: Seq[Setting[_]] = BuildSettings.projectSettings(assembly = true) ++ Seq(
-    mainClass in assembly := Some("com.socrata.balboa.agent.BalboaAgent"),
+  lazy val settings: Seq[Setting[_]] = BuildSettings.projectSettings ++ Seq(
     libraryDependencies <++= scalaVersion { libraries(_) },
-    jarName in assembly := s"${name.value}-assembly.jar",
```

Comment:
The way other services run is with `<service>-assembly-<version>.jar`. If we can use this standard it would be great.